draft-ietf-taps-interface.md

```diff
@@ -1555,11 +1573,11 @@ messageContext.add(parameter, value)
 
 messageData := "hel".octets()
 endOfMessage := false
-Connection.Send(messageData, messageContext, endOfMessage)
+Connection.Send(messageData, messageContext, reqRef?, endOfMessage)
```

Comment:
The messageContext parameter is exactly what this proposed message reference does. How about just adding more text to better explain the message context?

Reply:
I agree that the functionality of linking requests to responses may be a better done using the message context than using this parameter. Doing so would require to answer a few additional questions (see below).